LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2023 Fred Hutch Cancer Center
```

Comment:
In the past I've used

```suggestion
Copyright (c) YYYY Trevor Bedford and Richard Neher
```

but I'm no longer sure that is/was ever accurate? And in recent years, I wonder how HHMI's copyright interest (if any?) is involved here for those under their employ.

Maybe it should be 

```suggestion
Copyright (c) YYYY Nextstrain
```

and punt the question to "well, what's Nextstrain?".